common.cmake

```diff
@@ -56,8 +56,12 @@ function(nrfxlib_calculate_lib_path lib_path)
     elseif(DEFINED CONFIG_SOC_NRF54H20_CPURAD)
       set(arch_soc_dir ${arch_soc_dir}_cpurad)
     endif()
-  elseif(CONFIG_SOC_SERIES_BSIM_NRFXX)
-    set(arch_soc_dir "bsim_nrfxx")
+  elseif(CONFIG_SOC_SERIES_BSIM_NRF52X)
```

Comment:
We dont use this function in mpsl/sdc anymore but I guess it is ok to add it here as well.